app/pages/cif/create-project.tsx

```diff
@@ -3,29 +3,37 @@ import { withRelay, RelayProps } from "relay-nextjs";
 import { graphql, usePreloadedQuery } from "react-relay/hooks";
 import { createProjectQuery } from "__generated__/createProjectQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
-import updateFormChangeMutation, {
-  mutation,
-} from "mutations/FormChange/updateFormChange";
+import { mutation } from "mutations/FormChange/updateFormChange";
+import updateProjectRevisionMutation from "mutations/ProjectRevision/updateProjectRevision";
 import { useRouter } from "next/router";
-import ProjectForm from "components/Project/ProjectForm";
 import { Button } from "@button-inc/bcgov-theme";
 import Grid from "@button-inc/bcgov-theme/Grid";
 import { useMemo, useState } from "react";
 import useDebouncedMutation from "mutations/useDebouncedMutation";
 import SavingIndicator from "components/Form/SavingIndicator";
+import ProjecManagerForm from "components/Form/ProjectManagerForm";
+import ProjectForm from "components/Form/ProjectForm";
 
 const CreateProjectQuery = graphql`
   query createProjectQuery($id: ID!) {
     query {
       session {
         ...DefaultLayout_session
       }
-      formChange(id: $id) {
+      projectRevision(id: $id) {
```

Comment:
This file will eventually need to be refactored so that we have a single component to create, edit, or view a project revision